code/mob/new_player.dm

```diff
@@ -385,7 +385,14 @@ mob/new_player
 					logTheThing("debug", character, null, "<b>Late join:</b> added player to ticker.minds.")
 					ticker.minds += character.mind
 				logTheThing("debug", character, null, "<b>Late join:</b> assigned job: [JOB.name]")
-
+				//latejoin participation stuff
+#ifdef RP_MODE
+				var/max_participate_time = 80 MINUTES
+#endif
+				var/max_participate_time = 40 MINUTES
+				//if they have a ckey, joined before a certain threshold and the shuttle wasnt already on its way
+				if (character.mind.ckey && ticker.round_elapsed_ticks <= max_participate_time && !emergency_shuttle.online)
```

Comment:
ambig.
```suggestion
				if (character.mind.ckey && (ticker.round_elapsed_ticks <= max_participate_time) && !emergency_shuttle.online)
```

Reply:
thanks zew! i shoulda done this from the start